packages/nc-gui/store/tab.ts

```diff
@@ -10,14 +11,14 @@ function getPredicate(key: Partial<TabItem>) {
     (!('type' in key) || tab.type === key.type)
 }
 
-export const useTabs = createSharedComposable(() => {
+export const useTabs = defineStore('tabStore', () => {
   const tabs = ref<TabItem[]>([])
 
   const router = useRouter()
 
   const route = $(router.currentRoute)
 
-  const { bases, tables } = useProject()
+  const projectStore = useProject()
```

Comment:
need `storeToRefs` for `bases` and `tables`

Reply:
we are directly using it from `projectStore`